bungiesearch/signals.py

```diff
@@ -1,31 +1,55 @@
 from _collections import defaultdict
+
+from django.db.models import signals
+from importlib import import_module
+
 from . import Bungiesearch
 from .utils import update_index, delete_index_item
 
+def get_signal_processor():
+    signals = Bungiesearch.BUNGIE['SIGNALS']
+    if 'SIGNAL_CLASS' in signals:
+        module_str      = signals['SIGNAL_CLASS']
+        signal_module   = import_module(module_str)
+        signal_instance = signal_module()
```

Comment:
All this can be simplified in as follows:
```python
if 'SIGNAL_CLASS' in signals:
    signal_module = import_module(signals['SIGNAL_CLASS'])
else:
    signal_module = BungieSignalPorcessor
return signal_module()
```